addons/RiskOfRain/resource/reactivedrop_german.txt

```diff
@@ -3,195 +3,195 @@
-		"ror_itemDesc_red_freeze_grenades"		"Nearby frozen enemies take damage. Stacks range."
-		"[english]ror_itemDesc_red_freeze_grenades"		"Nearby frozen enemies take damage. Stacks range."
-		"ror_itemDesc_red_stim"		"Chance to drop a healing beacon on kill. Stacks beacon range."
-		"[english]ror_itemDesc_red_stim"		"Chance to drop a healing beacon on kill. Stacks beacon range."
-		"ror_itemDesc_red_tesla_trap"		"Stun nearby enemies every second. Stacks number of enemies that can be affected."
-		"[english]ror_itemDesc_red_tesla_trap"		"Stun nearby enemies every second. Stacks number of enemies that can be affected."
-		"ror_itemDesc_red_electrified_armor"		"Taking damage activates Electric Armor for 5 seconds. Stacks cooldown reduction."
-		"[english]ror_itemDesc_red_electrified_armor"		"Taking damage activates Electric Armor for 5 seconds. Stacks cooldown reduction."
-		"ror_itemDesc_red_mines"		"+5% CRIT chance. Burn enemies on critical hit. Burning enemies explode on death. Stacks damage."
-		"[english]ror_itemDesc_red_mines"		"+5% CRIT chance. Burn enemies on critical hit. Burning enemies explode on death. Stacks damage."
-		"ror_itemDesc_red_fist"		"All hits explode, dealing 60% TOTAL damage. Stacks range."
-		"[english]ror_itemDesc_red_fist"		"All hits explode, dealing 60% TOTAL damage. Stacks range."
-		"ror_itemDesc_red_grenades"		"Killing 4 enemies in 7 seconds fires a high damage grenade at the nearest enemy. Stacks damage."
-		"[english]ror_itemDesc_red_grenades"		"Killing 4 enemies in 7 seconds fires a high damage grenade at the nearest enemy. Stacks damage."
-		"ror_itemDesc_red_night_vision"		"Chance to reanimate killed enemies into ghost allies. Stacks ghost lifespan."
-		"[english]ror_itemDesc_red_night_vision"		"Chance to reanimate killed enemies into ghost allies. Stacks ghost lifespan."
-		"ror_itemDesc_red_smart_bomb"		"All item-spawned rockets spawn 2 more times. Stacks bonus damage to rockets."
-		"[english]ror_itemDesc_red_smart_bomb"		"All item-spawned rockets spawn 2 more times. Stacks bonus damage to rockets."
-		"ror_itemDesc_red_gas_grenades"		"Heal every second. Infestations are instantly cured when below 50% health. Stacks heal amount."
-		"[english]ror_itemDesc_red_gas_grenades"		"Heal every second. Infestations are instantly cured when below 50% health. Stacks heal amount."
-		"ror_itemDesc_red_blink"		"All random effects are rolled again for a favorable outcome. Stacks number of rerolls."
-		"[english]ror_itemDesc_red_blink"		"All random effects are rolled again for a favorable outcome. Stacks number of rerolls."
-		"ror_itemDesc_red_jump_jet"		"Hitting an enemy permanently increases the damage the enemy takes next time. Stacks increased damage."
-		"[english]ror_itemDesc_red_jump_jet"		"Hitting an enemy permanently increases the damage the enemy takes next time. Stacks increased damage."
+"ror_translationCredit"		"Englische Übersetzungen von ModdedMarionette."
```

Comment:
this should say that you did the German translation

Reply:
should adding comment with explanation help? //replace language name and author's name with your language and name.